policyengine_us/reforms/congress/tlaib/economic_dignity_for_all_agenda/edaa_end_child_poverty_act.py

```diff
@@ -0,0 +1,237 @@
+        entity = SPMUnit
+        label = "Benefits"
+        definition_period = YEAR
+        unit = USD
+
+        def formula(spm_unit, period, parameters):
+            BENEFITS = [
+                "social_security",
+                "ssi",
+                "ma_state_supplement",
+                "ca_cvrp",
+                "co_ccap_subsidy",
+                "co_state_supplement",
+                "co_oap",
+                "snap",
+                "wic",
+                "free_school_meals",
+                "reduced_price_school_meals",
+                "spm_unit_broadband_subsidy",
+                "spm_unit_energy_subsidy",
+                "tanf",
+                "high_efficiency_electric_home_rebate",
+                "residential_efficiency_electrification_rebate",
+                "unemployment_compensation",
+                "ak_energy_relief",
+                "basic_income",
+                "ny_drive_clean_rebate",
+            ]
+
+            # Add ECPA child benefit
+            BENEFITS.append("ecpa_child_benefit")
+
+            if parameters(period).gov.contrib.ubi_center.flat_tax.deduct_ptc:
+                BENEFITS.append("aca_ptc")
+            if not parameters(period).gov.hud.abolition:
+                BENEFITS.append("spm_unit_capped_housing_subsidy")
+
+            return add(spm_unit, period, BENEFITS)
+
+    class income_tax_refundable_credits(Variable):
+        value_type = float
+        entity = TaxUnit
+        definition_period = YEAR
+        label = "federal refundable income tax credits"
+        unit = USD
+
+        def formula(tax_unit, period, parameters):
+            # Get standard refundable credits, excluding EITC and refundable CTC (ECPA replaces them)
+            standard_credits = parameters(period).gov.irs.credits.refundable
+
+            CREDITS = [
+                c
+                for c in standard_credits
+                if c not in ["eitc", "refundable_ctc"]
+            ]
+
+            base_credits = add(tax_unit, period, CREDITS) if CREDITS else 0
+
+            # Add ECPA credits
+            ecpa_filer = tax_unit("ecpa_filer_credit", period)
+            ecpa_adult_dep = add(
+                tax_unit, period, ["ecpa_adult_dependent_credit"]
+            )
+            return base_credits + ecpa_filer + ecpa_adult_dep
+
+    class income_tax_non_refundable_credits(Variable):
+        value_type = float
+        entity = TaxUnit
+        definition_period = YEAR
+        label = "federal non-refundable income tax credits"
+        unit = USD
+
+        def formula(tax_unit, period, parameters):
+            # Get the base list of non-refundable credits from parameters
+            p = parameters(period).gov.irs.credits
+            base_credits = list(p.non_refundable)
+
+            # ECPA removes non_refundable_ctc from the list
+            CREDITS = [c for c in base_credits if c != "non_refundable_ctc"]
+
+            return add(tax_unit, period, CREDITS)
```

Comment:
Does it completely repeal the non refundable CTC? I wonder whether we should use the `neutralize_variable` instead since there are some minor interactions with state level programs. 

Basically the question is whether the non refundable CTC value should e 0 or of it should still be computed but not added to the list of non-refundable credits